project.clj

```diff
@@ -16,8 +16,8 @@
                              [lein-cljsbuild "1.1.6"]]
                    :jvm-opts ^:replace ["-server"]
                    ;:global-vars {*warn-on-reflection* true}
-                   :dependencies [[org.clojure/clojure "1.9.0-alpha15"]
-                                  [org.clojure/clojurescript "1.9.518"]
+                   :dependencies [[org.clojure/clojure "1.9.0-alpha17"]
+                                  [org.clojure/clojurescript "1.9.562"]
```

Comment:
With the new  `resolve`, it needs some work on `spec-tools`. Would you like to do the honors? Also, there were some patches to spec that might not yet be in `clojure.spec`?